sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -3104,6 +3117,90 @@ atomic_update_impl(T *p, simd<Toffset, N> offsets, simd_mask<N> pred) {
   return lsc_format_ret<T>(Tmp);
 }
 
+/// USM pointer atomic.
+/// Supported platforms: DG2, PVC
+/// VISA instruction: lsc_atomic_<OP>.ugm
+///
+/// @tparam Op is operation type.
+/// @tparam T is element type.
+/// @tparam N is the number of channels (platform dependent).
+/// @tparam DS is the data size.
+/// @tparam L1H is L1 cache hint.
+/// @tparam L2H is L3 cache hint.
```

Comment:
```suggestion
/// @tparam L2H is L2 cache hint.
```

Reply:
Fixed in https://github.com/intel/llvm/pull/11862/commits/687650629fde42caf2821cdcd5d7f952f6fafd8b.